submissions/motunrayoIlawole/challenge1-arrays/question1.js

```diff
@@ -0,0 +1,47 @@
+function solution(strs) {
```

Comment:
Interesting solution, works very fine, However `undefined` is returned for the input case `[""]`. 
The expected output is `""`. I think it might be because your `commonPrefix` array could be empty at some point.

The time complexity is actually O(nlogn) due to the sorting on line 39. The major loop is actually an O(mn) loop. 
m being the length of the longest string in the array. But that would probably be less than the O(logn), so you could round it out to O(nlogn)